sphinx_click/ext.py

```diff
@@ -1,12 +1,23 @@
 import re
 import traceback
 import warnings
+from typing import Iterable
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
```

Comment:
Rather than importing the individual types, can you import `typing` and use that. i.e.

```python
import typing as ty

from click import core

# ...

def _get_help_record(opt: core.Option) -> ty.Tuple[str, str]:
    # ...
```

Reply:
What's the purpose of aliasing `Tuple` to `ty.Tuple`?  
I don't mind that it's longer, but I fear that using a custom  name for these common types will hurt readability with those unfamiliar with this codebase.

Reply:
It's not aliasing, at least not for the classes - I tend to prefer importing modules rather than the classes themselves because it's less noisy. You could drop the aliasing of the module (so `typing.Tuple` but that seems unnecessarily verbose